src/utilities/helpers/helpers.js

```diff
@@ -0,0 +1,4 @@
+export const eventsCleaner = (events) => {
+  const eventsArray = events._embedded.events;
+  return eventsArray;
```

Comment:
eventsCleaner is returning an Array?  In App state, you have events set as an empty object.